src/api/søknadApi.ts

```diff
@@ -0,0 +1,74 @@
+import apiClient, { ApiClientResult } from './apiClient';
+import { Nullable } from '~lib/types';
+
+export interface Søknad {
+    personopplysninger: {
+        aktørid: string;
+        fnr: string;
+        fornavn: string;
+        mellomnavn: Nullable<string>;
+        etternavn: string;
+        telefonnummer: string;
+        gateadresse: string;
+        postnummer: string;
+        poststed: string;
+        bruksenhet: string;
+        bokommune: string;
+        statsborgerskap: string;
+    };
+    uførevedtak: {
+        harUførevedtak: boolean;
+    };
+    flyktningsstatus: {
+        registrertFlyktning: boolean;
+    };
+    oppholdstillatelse: {
+        harVarigOpphold: boolean;
+        utløpsdato: Nullable<string>;
+        søktOmForlengelse: boolean;
+    };
+    boforhold: {
+        borFastINorge: boolean;
+        borPåFolkeregistrertAdresse: boolean;
+        delerBolig: boolean;
+        borSammenMed: Nullable<Array<{ fnr: string; navn: string }>>;
+        delerBoligMed: Nullable<Array<{ fnr: string; navn: string }>>;
```

Comment:
Kanskje enklere å bare bruke `Array<{ fnr: string; navn: string }>` for disse (og de andre listene)? Føler det er litt redundant å ha dem som `Nullable` også.